layout-server/src/main/java/org/finos/vuu/layoutserver/model/Layout.java

```diff
@@ -0,0 +1,26 @@
+package org.finos.vuu.layoutserver.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import lombok.Data;
+import java.util.UUID;
+
+@Data
+@Entity
+public class Layout {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    private String definition;
```

Comment:
This might be more appropriate as a `JsonNode` than a string (import from Jackson Databind). You'll have to annotate the field with `@Lob`, but it should avoid the risk of malformed JSON getting into the system.

Reply:
@pling-scottlogic Did you get any further with this? Is it worth discussing offline?

Reply:
Have a look at what I've done [here](https://github.com/finos/vuu/commit/cfbbe5c27fedb10352c2021b967ceefa93348b8e) - I'm using `JsonNode` for data transfer, but still using a string (with a JSON column definition) on the entity. It could probably do with a bit more refining, considering the string manipulation I'm having to do.

Reply:
Ticket created: https://github.com/orgs/ScottLogic/projects/21/views/1?pane=issue&itemId=41812297